### PR TITLE
Update changelog for v0.18.0

### PR DESCRIPTION
**Pull request by ly015**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v0.18.0 (01/09/2021)
+
+**Bug Fixes**
+
+- Fix redundant model weight loading in pytorch-to-onnx conversion ([\#850](https://github.com/open-mmlab/mmpose/pull/850)) @ly015
+- Fix a bug in update_model_index.py that may cause pre-commit hook failure([\#866](https://github.com/open-mmlab/mmpose/pull/866))@ly015
```
Comment (jin-s13) [inline docs/changelog.md:8]: add a space before @

File: configs/face/2d_kpt_sview_rgb_img/topdown_heatmap/cofw/hrnetv2_cofw.md [hunk 2]
```diff
@@ -33,6 +33,4 @@
 
 </details>
 
-</details>
```
Comment (innerlee) [inline configs/face/2d_kpt_sview_rgb_img/topdown_heatmap/cofw/hrnetv2_cofw.md:36]: Does this belong to another pr?

Comment (ly015) [reply configs/face/2d_kpt_sview_rgb_img/topdown_heatmap/cofw/hrnetv2_cofw.md:36]: Seems so. will fix it.
